src/faldbt/lib.py

```diff
@@ -192,9 +191,9 @@ def _get_target_relation(
                 target_name = target.alias
```

Comment:
If we don't use `target_name`, then shouldn't we remove this part of the code as well?

Reply:
thanks!